native/cocos/bindings/jswrapper/v8/Class.cpp

```diff
@@ -107,27 +102,43 @@ bool Class::init(const ccstd::string &clsName, Object *parent, Object *parentPro
         _parentProto->incRef();
     }
 
-    _ctor = ctor;
+    _constructor = ctor;
 
-    v8::FunctionCallback ctorToSet = _ctor != nullptr ? _ctor : invalidConstructor;
+    v8::FunctionCallback ctorToSet = _constructor != nullptr ? _constructor : invalidConstructor;
```

Comment:
Distinguish `_ctor` & `_constructor`, `_constructor` is the real JS constructor while `_ctor` is a method implemented in `xxx.jsb.ts`, which invoked at the end of `JSB object construction`.